openquake/qa_tests_data/classical/case_5/job.ini

```diff
@@ -17,7 +17,7 @@ number_of_logic_tree_samples = 0
 
 # km
 rupture_mesh_spacing = 0.01
-complex_fault_mesh_spacing = 0.01
+complex_fault_mesh_spacing = 0.1
```

Comment:
Please check if you might want to change also `rupture_mesh_spacing = 0.1` as in other cases